utils/kytConfig.js

```diff
@@ -4,12 +4,12 @@
 const path = require('path');
 const shell = require('shelljs');
 const mergeAll = require('ramda').mergeAll;
+const logger = require('../cli/logger');
```

Comment:
Take a look at logger, it no longer references kytConfig. This way logger stays 'pure' and the logic around logging or not is embedded in the app.